lib/script.js

```diff
@@ -0,0 +1,70 @@
+/**** To change time from UNIX to normal ***/
+// var time = new Date(timestamp*1000);
+
+var weather_data = [];
+
+$(document).ready(function() {
+    ShowCurrentTime();
+    getWeather("denver", "co");
+    pauseThenLoadWeather();
+})
+
+const delay = ms => new Promise(res => setTimeout(res, ms));
+const pauseThenLoadWeather = async () => {
+    document.getElementById('fahrenheit-symbol').innerHTML = ""
+    await delay(2000);
+    document.getElementById('fahrenheit-symbol').innerHTML = "F";
+    displayCurrentWeather();
+    document.querySelector('.lds-roller').style.display = 'none';
+  };
+
+function ShowCurrentTime() {
+    var time = moment().format('h:mm a');
+    document.getElementById('current-weather__time').innerHTML = time;
+
+    setInterval(() => {
+        document.getElementById('current-weather__time').innerHTML = time;
+      }, 1000);
+} 
+
+function getWeather(city, state) {
+    fetch(`https://rocky-cliffs-95700.herokuapp.com/api/v1/forecast?location=${city,state}`)
+    .then(response => response.json())
+    .then(result => weather_data.push(result));
```

Comment:
I would add a .catch() function here to display an error to the console in case something goes wrong with with your API or the js. Other than that, this function looks good!

Reply:
nice CATCH lol - thanks man. will implement!